README.md

```diff
@@ -126,6 +130,84 @@ Response:
   Body: <preimage_bytes>
 ```
 
+#### Admin Routes
+
+The proxy provides administrative endpoints to control runtime behavior. By default, these endpoints are disabled and must be explicitly enabled through configuration.
```

Comment:
let's add a disclaimer specifying that this should never be publicly accessible - then again the namespace `/admin` already infers that